tests/tizen_capi/unittest_tizen_sensor.cc

```diff
@@ -18,6 +18,21 @@
 #include <gst/gst.h>
 #include <nnstreamer.h>
 #include <nnstreamer-capi-private.h>
+#include <unittest_util.h>
+
+static const uint TEST_TIME_OUT_TIZEN_SENSOR = 10000; /* timeout occur after 10 seconds */
+
+#define wait_for_start(handle, state, status) \
+do {\
+  int counter = 0;\
+  while ((state == ML_PIPELINE_STATE_PAUSED || \
```

Comment:
maybe condition for state playing is (state != ML_PIPELINE_STATE_PLAYING)

Reply:
Thank you. Updated!